input/templates/my-account-change-password.hbs

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {{> common/htmlhead}}
+</head>
+
+<body>
+  <div class="darkbg hidden"></div>
+  {{> common/header}}
+  <div class="my-account">
+    <div class="container">
+      <div class="my-account-title">
+        <span class="my-account-title-text icon-user">{{i18n "my-account:myAccount.title"}}</span>
+      </div>
+      <div class="row">
+        <div class="col-sm-9">
+          <div class="my-account-content">
+            <div class="row">
+              <div class="col-sm-3">
+                {{> myaccount/my-account-sidebar positionSeven=true}}
+              </div>
+              <div id="my-account-desktop-content" class="col-sm-9">
+                <div class="change-password">
+                  <div class="change-password-wrapper">
+                    <div class="personal-details-title">
+                      <span>{{i18n "my-account:changePassword.changePasswordTitle"}}</span>
+                      <span class="pull-right required-text">{{i18n "main:form.required"}}*</span>
+                    </div>
+                    <form action="">
+                      <div class="row">
+                        <div class="col-sm-6">
+                          <div class="form-sections">
+                            <span class="form-labels">{{i18n "my-account:changePassword.labelTxt"}}*</span><br>
+                            <input class="form-inputs" type="text">
+                          </div>
+                        </div>
+                        <div class="col-sm-6">
+                          <div class="form-sections">
+                            <span class="form-labels">{{i18n "my-account:changePassword.confirmLabelTxt"}}*</span><br>
+                            <input class="form-inputs" type="text">
+                          </div>
+                        </div>
+                      </div>
+                    </form>
+                    <hr class="light-grey-hr">
+                    <div class="personal-details-edit-btn">
+                      <span>
+                        <button class="update-btn password-change-btn">{{i18n "my-account:changePassword.changeBtn"}}</button>
+                      </span>
+                    </div>
+                  </div>
+                  <div class="password-feedback-wrapper">
+                    <br /><br />
+                    <span>{{i18n "my-account:changePassword.feedbackMessage"}}</span>
```

Comment:
Ah you need to include the `global-errors` template above or below the `global-messages` (I mean, we need both).